inst/rmarkdown/resources/js/date-fiche.js

```diff
@@ -0,0 +1,11 @@
+let message = new String();
+let today = new Date();
+let dd = String(today.getDate()).padStart(2, '0');
+let mm = String(today.getMonth() + 1).padStart(2, '0'); //January is 0
+let yyyy = today.getFullYear();
+
+today = `${dd}/${mm}/${yyyy}`;
+
+message = 'Source: www.utilitr.org, date: ' + today;
+
+document.documentElement.style.setProperty('--date-fiche', `'${message}'`);
```

Comment:
Ca ne sert à rien de polluer avec tous ces objets globaux

```suggestion
{
  let dd = String(today.getDate()).padStart(2, '0');
  let mm = String(today.getMonth() + 1).padStart(2, '0'); //January is 0
  let yyyy = today.getFullYear();
  let today = `${dd}/${mm}/${yyyy}`;
  let message = 'Source: www.utilitr.org, date : ' + today;

  document.documentElement.style.setProperty('--date-fiche', `'${message}'`);
}
```